backend/schemas/sections/FederalAwards.schema.json

```diff
@@ -142,7 +138,7 @@
         },
         {
           "const": "10.128",
-          "description": "Heirs\u2019 Property Relending Program"
```

Comment:
All of these `\ufffd` replacements look rather suspicious…

Reply:
yeah the assistance listings files from SAM.gov have some weird encoding issues - I'll try to get some clarity on that before marking this as ready